### PR TITLE
Add analysis page 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, there is no Cookie modal.
- Currently, there is no dedicated analyisis page.
- Currently, the 404 page does not have HTML headers.
- Currently, the contact form is not GDPR safe.
- Currently, there can be a bug where a value in the signet exeeds 100.

**What is the new behavior (if this is a feature change)?**
- A cookie modal has been added.
- A dedicated analysis page has been added.
- The 404 page now has HTML headers.
- The contact form has been made GDPR safe by adding a checkbox (opt-in).
- The maximum value of the signet has been set to 100.

**Other information**:
- The version has been updated to v1.3.0.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks good to meh.

File: src/components/pages/AnalysisPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+                <MDBCard className="mb-3">
+                  <MDBCardBody className="text-left">
+                    <p className="lead font-weight-bold">
+                      Wieso ist eine starke Online-Präsenz so wichtig?
+                    </p>
+                    <p className="feature lead">
+                      <MDBIcon icon="check" className="green-text" />
+                      Erhöhte Reichweite
+                      <small className="text-muted">
+                        Eine Website macht Kunden über Dein Unternehmen
```
Comment (LucaAllmaier) [inline src/components/pages/AnalysisPage/index.jsx]: auf Dein Unternehmen

File: src/components/pages/AnalysisPage/index.jsx [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+  };
+
+  onCheckChange = (e) => {
+    this.setState({
+      [e.target.name]: e.target.checked,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
```
Comment (schettn) [inline src/components/pages/AnalysisPage/index.jsx:56]: add newline

File: src/components/pages/AnalysisPage/index.jsx [hunk 3]
```diff
@@ -0,0 +1,318 @@
...
+                    Über unser innovatives Kunden-Informations-System (KISy)
+                    kannst Du einfach Unternehmen miteinander vergleichen.
+                  </small>
+                </p>
+                <p className="feature lead">
+                  <MDBIcon icon="award" className="indigo-text" />
+                  Vorher / Nachher
+                  <small className="text-muted">
+                    Der Vergleich macht sich sicher! Wie war der Stand eines
+                    Unternehmens bevor und nacherdem wir deren Online-Präsenz
```
Comment (LucaAllmaier) [inline src/components/pages/AnalysisPage/index.jsx]: nachdem - typo

**Review by schettn — CHANGES_REQUESTED** [review]
Should be ok after a change

**Review by LucaAllmaier — APPROVED** [review]
Looks good to merge.

**Review by schettn — APPROVED** [review]
LGTM!

**Pull request merged**